### PR TITLE
Ball speeds up every 5 seconds

### PR DESCRIPTION
**Pull request by Yugo0**

### CODE REVIEW

File: source/dagger/gameplay/ping_pong/pingpong_ball.h [hunk 1]
```diff
@@ -10,6 +13,9 @@ namespace ping_pong
     struct PingPongBall
     {
         Vector3 speed{ 0, 0, 0 };
+        
+        std::chrono::seconds changeSpeedPeriod{ 5 };
+        TimePoint lastSpeedChange{ dagger::Engine::Instance().CurrentTime() };
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/pingpong_ball.h]: looks fine. 

But you can implement it in more usual way:
have a float timer = 5 inside PingPongBall
and do timer -= DeltaTime each frame.
and check if it's < than a 0. 
That way you won't need to work with chrono library inside gameplay classes.

**Review by DPrivalov — APPROVED** [review]

File: source/dagger/gameplay/ping_pong/pingpong_ball.h (hunk 1 above)
Comment (gavrilovmiroslav) [reply source/dagger/gameplay/ping_pong/pingpong_ball.h]: Hmmm, our Engine's [delta time](https://github.com/UbiBelETF/dagger/blob/b5ca5c344b1be7513abd06af5fdec1b82c2143bb/source/dagger/core/engine.h#L61) function might help here! I don't think we have it documented anywhere officially... :)

File: source/dagger/gameplay/ping_pong/pingpong_ball.h [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 #include "core/system.h"
 #include "core/core.h"
+#include "core/engine.h"
+
+#include <chrono>
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/pingpong_ball.h]: remove this one pls

**Comment by DPrivalov** [comment]
waiting for 2 teammates to review and it's good to go

**Review by kostaDimitrijevic — APPROVED** [review]

**Review by kostaDimitrijevic — APPROVED** [review]

**Review by smiljanaspasic — APPROVED** [review]

**Pull request merged**